package.json

```diff
@@ -4,14 +4,17 @@
   "private": true,
   "homepage": "https://st6.io/meme",
   "dependencies": {
+    "is-image": "3.0.0",
     "react": "16.8.6",
+    "react-addons-css-transition-group": "15.6.2",
```

Comment:
This is the legacy CSS transition add-on. It will be better to use the newer (and officially supported) version: https://github.com/reactjs/react-transition-group